azure-iot-device/azure/iot/device/iothub/sync_clients.py

```diff
@@ -576,3 +610,15 @@ def invoke_method(self, method_params, device_id, module_id=None):
         invoke_method_response = handle_result(callback)
         logger.info("Successfully invoked method")
         return invoke_method_response
+
+    @property
+    def on_message_received(self):
+        """The handler function that will be called when a message is received.
```

Comment:
>The handler function that will be called when a message is received. [](start = 11, length = 68)

should say "when an input message is received".  If this is going to be public doc, then you should mention the name of the message attribute that holds the input name.

Reply:
I'm not sure - technically this function could return any type of message the client receives. Right now that's only input messages, sure, but an input message IS just a message.

Also I think documentation re: details of the message should be captured on the doc for the Message object rather than here.

---
In reply to: [480408673](https://github.com/Azure/azure-iot-sdk-python/pull/651#discussion_r480408673) [](ancestors = 480408673)